gettsim/parameters/ges_rente.yaml

```diff
@@ -730,30 +730,84 @@ regelaltersgrenze:
+      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger,
+      wenn keine Ausnahmeregelungen erfüllt sind.
     en:
       Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
       claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
-      the pension claim is higher or lower.
+      the pension claim is higher or lower, unless special conditions apply.
   reference: § 35 Satz 2 SGB VI
   note:
     de: >-
       Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
-      implementiert.
+      implementiert. Bis einschließlich Jahrgang 1946 ist NRA 65, danach steigt es bis Jahrgang 1964.
+      Ab einschließlich Jahrgang 1964 gilt NRA 67. (todo jahrgänge vor 1946)
     en:
   type: piecewise_linear
   2002-01-01:
     0:
       lower_threshold: -inf
-      upper_threshold: 1947
+      upper_threshold: 1946
       rate_linear: 0
       intercept_at_lower_threshold: 65
     1:
-      lower_threshold: 1947
-      upper_threshold: 1971
+      upper_threshold: 1958
       rate_linear: 0.083333333
     2:
-      lower_threshold: 1971
+      upper_threshold: 1964
+      rate_linear: 0.16666666
+    3:
+      upper_threshold: inf
+      rate_linear: 0
+
+altersgrenze_langjahrig:
+  name:
+    de: Eintrittsalter für langjaehrig versicherte - abschlagsfrei
+    en: Entry age for long term insured (35 years) without deductions
+  description:
+    de:
+      Stückweise lineare Funktion, die die Regelaltersgrenze angibt, bei der das Individuum mit seinem vollen Rentenanspruch in Rente
+      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger,
+      wenn keine Ausnahmeregelungen erfüllt sind.
+    en:
+      Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
+      claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
+      the pension claim is higher or lower, unless special conditions apply.
+  reference: § 35 Satz 2 SGB VI
+  note:
+    de: >-
+      Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
+      implementiert. Kohorten vor 1937 konnten mit 63 abschlagsfrei in Rente gehen.
+      Ab Geburtsdatum Januar 1937 wurde fuer jeden Monat, den die Person spaeter geboren wurde das
+      Eintrittsalter um 1 Monat erhoeht bis Geburtsdatum Dezember 1938: Eintrittsalter 65. Dieses
+      Eintrittsalter galt bis einschliesslich Jahrgang 1948. Ab Geburtsdatum Maerz 1949 entspricht die
+      Altersgrenze der Regelaltersgrenze, also 65 Jahre, 3 Monate. Fuer Geburtsdatum 1/1949 ist sie
+      65, 1 Monat. Fuer 2/1949 65, 2 Monate.
+    en:
+  type: piecewise_linear
+  2002-01-01:
```

Comment:
Again, use earliest date where this makes sense. Might need to distinguish men and women?

Reply:
is (re)moved now to another issue